static/app/components/projects/useProjectCreationAccess.spec.tsx

```diff
@@ -26,6 +27,19 @@ describe('ProjectCreationAccess', function () {
     expect(result.current.canCreateProject).toBeTruthy();
   });
 
+  it('fails for members if org has team-roles but disabled allowMemberProjectCreation', function () {
+    const experiment_org = OrganizationFixture({
+      access: ['org:read', 'team:read', 'project:read'],
+      features: ['team-roles'],
+      allowMemberProjectCreation: false,
+    });
+
+    const {result} = renderHook(useProjectCreationAccess, {
+      initialProps: {organization: experiment_org},
+    });
```

Comment:
this isn't a hook so you can just unit test it normally
```suggestion
    const result = useProjectCreationAccess{{organization: experiment_org})
```